client/js/components/snippets/snippet.jsx

```diff
@@ -70,7 +70,7 @@ export default class Snippet extends React.Component {
     let author = (this.props.user ? this.props.user.name : (currentUser ? currentUser.name : 'author') );
     let enabled = this.checkRatingAbility();
 
-    const ratings = this.props.id ? <RatingForm key={this.props.id} snippetId={this.props.id} snippet={this.props} style={{right: 0, margin: '10px'}} enabled={enabled}/>: '';
+    let ratings = this.props.id ? <RatingForm key={this.props.id} snippetId={this.props.id} snippet={this.props} style={{right: 0, margin: '10px'}} enabled={enabled}/>: '';
```

Comment:
I think const it's not necessary here so I've made it consistent with rest of variables.